convert_model/convert_bert_to_torch.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+from argparse import ArgumentParser
+
+import tensorflow as tf
+import torch
+from models.torch.bert import BertConfig as TorchBertConfig
+from models.torch.bert import BertModel as TorchBertModel
+
+arg_parser = ArgumentParser('Convert Google BERT model to PyTorch BERT model.')
+arg_parser.add_argument('--ckpt_path', type=str, required=True, help="Checkpoint path of the model to be converted")
+arg_parser.add_argument('--config_path', type=str, required=True, help="Config path of the model to be converted")
+arg_parser.add_argument('--save_dir', type=str, default=".", help="Save path of the config path")
+args = arg_parser.parse_args()
```

Comment:
언더바("_") 대신에 대쉬("-")를 쓰는 게 국룰이라고 하더라구요 (욱재피셜) ㅋㅋ
```suggestion
arg_parser.add_argument('--ckpt-path', type=str, required=True, help="Checkpoint path of the model to be converted")
arg_parser.add_argument('--config-path', type=str, required=True, help="Config path of the model to be converted")
arg_parser.add_argument('--save-dir', type=str, default=".", help="Save path of the config path")
args = arg_parser.parse_args()
```

Reply:
하지만 그냥 머지할래요 ㅋㅋ @Kyeongpil